azerothcore/azerothcore-wotlk
# [DB/Quest] Gaining Even More Acceptance

- QUEST: Gaining Even More Acceptance

##### CHANGES PROPOSED:

-  make quest repeteable !
-  add 25 reputation in database


###### ISSUES ADDRESSED:


Closes 


##### TESTS PERFORMED:


- in my server , one player reported this  !

https://www.youtube.com/watch?v=xn_MmJC5My4 video guide !



##### HOW TO TEST THE CHANGES:


- Go to NPC : Master Smith Burninate <The Thorium Brotherhood>
- Take Quest and complete material and see not repeteable


- In column no exist value 25 reputation



##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
